rpc/v1/openapi.yaml

```diff
@@ -130,110 +218,1622 @@ paths:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Account"
+        "400":
+          description: Bad request
+          content:
+            text/plain:
+              example: "description of failure"
+        "500":
+          description: An error occurred while retrieving the account data at the specified height
+          content:
+            text/plain:
+              example: "description of failure"
+  /v1/query/accounts:
+    post:
+      tags:
+        - query
+      summary: Returns the paginated data of all accounts specified at a given height
+      requestBody:
+        description: Request all account data at the specified height, height = 0 is used as the latest; Max per_page=1000
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/QueryHeightPaginated"
+            example:
+              height: 0
+              page: 1
+              per_page: 50
+        required: true
+      responses:
+        "200":
+          description: Returns account data for all accounts at the specified height
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QueryAccountsResponse"
+        "400":
+          description: Bad request
+          content:
+            text/plain:
+              example: "description of failure"
+        "500":
+          description: An error occurred while retrieving accounts at the specified height
+          content:
+            text/plain:
+              example: "description of failure"
+  /v1/query/account_txs:
+    post:
+      tags:
+        - query
+      summary: Returns all transactions sent by the specified address paginated
+      requestBody:
+        description: Returns all transactions sent by the address provided; Max per_page=1000, can be sorted either asc or desc (default)
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/QueryAddressPaginated"
+            example:
+              address: da034209758b78eaea06dd99c07909ab54c99b45
+              page: 1
+              per_page: 1000
+              sort: desc
+        required: true
+      responses:
+        "200":
+          description: Returns the transaction list for the account
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QueryAccountTxsResponse"
+        "400":
+          description: Bad request
+          content:
+            text/plain:
+              example: "description of failure"
+        "500":
+          description: An error occurred while retrieving the transactions for the account
+          content:
+            text/plain:
+              example: "description of failure"
+  /v1/query/all_chain_params:
     get:
       tags:
         - query
       summary: Returns the current values of all governance parameters
```

Comment:
Is this the intended summary for all_chain_params?

Reply:
Yeah, I think this is pretty clear.